contribution.md

```diff
@@ -0,0 +1,237 @@
+
+> **NOTE 1**: due OpenVINO&trade; paradigms, mean and scale values are built-in converted model.
+
+> **NOTE 2**: due OpenVINO&trade; paradigms, if model take colored image as input, color channel order supposed to be `BGR`.
+
+*After this step you`ll get **conversion parameters** for Model Optimizer.*
+
+## Demo
+
+Demo will show main idea of how work with your model. If your model solves one of the supported by Open Model Zoo task, try find appropriate option from [demos](https://docs.openvinotoolkit.org/latest/_demos_README.html) or [samples](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_Samples_Overview.html).
+
+If appropriate demo or sample are absent, you must provide your own demo (C++ or Python). Demo's input requires next options:
+```
+    -i "<path>"              Optional. Path to a input file or directory (for multiple inferences). By default input must be generated randomly.
+    -m "<path>"              Required. Path to an .xml file with a trained model
+    -d "<device>"            Optional. Target device for model inference. Usually CPU and GPU. By default CPU
+    -no_show                 Optional. Do not show inference result.
+```
+Also you can add all necessary parameters for inference your model.
+
+## Accuracy validation
+
+To run accuracy validation, use [Accuracy Checker](./tools/accuracy_checker/README.md) tool, provided with repository. Doing this very simple if model task from supported. You must only create accuracy validation configuration file in this case. 
+
+When the configuration file is ready, you must run Accuracy Checker to obtain metric results. If they match your results, that means  conversion was fully successful and Accuracy Checker fully supports your model, metric and dataset. If no - recheck [conversion][#model-conversion] parameters or validation configuration.
+
+*After this step you will get accuracy validation configuration file - **<model_name>.yml***
+
+## Configuration file
+
+Models configuration file contains information about model: what it is, how to download it and how to convert it to IR format. This information must be specified in `model.yml` file, which must be in the models subfolder. Let look closer to the file content.
+
+**`description`**
+
+This tag must contain description of model.
+
+**`task_type`**
+
+This tag describes on of the task that model solves:
+- `action_recognition`
+- `classification`
+- `detection`
+- `face_recognition`
+- `head_pose_estimation`
+- `human_pose_estimation`
+- `image_processing`
+- `instance_segmentation`
+- `object_attributes`
+- `optical_character_recognition`
+- `semantic_segmentation`
+
+If your model solves another task, you can freely add it with modification of [tools/downloader/common.py](tools/downloader/common.py) file list `KNOWN_TASK_TYPES`
+
+**`files`**
+
+You must describe all files, which must be downloaded, in this section. Each file must is described in few tags:
```

Comment:
, which need to be downloaded, in this section. Please describe each file in few tags: